Updated translation and added tooltips (#382)
xcube-dev/xcube-viewer: public/docs/user-variables.de.md

> Ein _Nutzer-Variable_ ist eine Variable, die durch einen _Namen_, einen 

Reviewer: ```suggestion
Eine _Nutzer-Variable_ ist eine Variable, die durch einen _Namen_, einen 
```

_Titel_, _Einheiten_ und einen algebraischen _Ausdruck_ definiert wird, der 
verwendet wird, um die Arraydaten der Variable zu berechnen. Nutzer-Variablen 

> werden dem aktuell ausgewählten Datensatz hinzugefügt und ihre Ausdrücke 

Reviewer: ```suggestion
werden dem aktuell ausgewählten Datensatz hinzugefügt und deren Ausdrücke 
```

**Name**: Ein Name, der innerhalb der Variablen des ausgewählten Datensatzes 
eindeutig ist. Der Name muss mit einem Buchstaben beginnen, gefolgt von 
Buchstaben oder Ziffern.

**Titel**: Optionaler Anzeigename der Variablen in der Benutzeroberfläche.

**Einheiten**: Optionale physikalische Einheiten der berechneten Datenwerte. 
Zum Beispiel werden Einheiten verwendet, um Zeitreihen zu gruppieren.

**Ausdruck**: Ein algebraischer Ausdruck, der verwendet wird, um die Datenwerte 
der Variablen zu berechnen. Die Syntax entspricht der von [Python-Ausdrücken](https://docs.python.org/3/reference/expressions.html). 
Der Ausdruck kann die folgenden Namen referenzieren:
- die Datensätze des aktuellen Datensatzes;
- die numpy-Konstanten `e`, `pi`, `nan`, `inf`;
- alle [numpy ufunc](https://numpy.org/doc/stable/reference/ufuncs.html) Funktionen;
- die [`where`](https://docs.xarray.dev/en/stable/generated/xarray.where.html) Funktion.

Die Mehrheit der numerischen und logischen Operatoren von Python wird 
unterstützt, jedoch können die logischen Operatoren `and`, `or` und `not` nicht 
mit Array-Variablen verwendet werden, da sie boolesche Werte als Operanden 

> erfordern. Verwenden Sie stattdessen die bitweisen Operatoren `&`, `|`, `~` oder 
> die entsprechenden Funktionen `logical_and()`, `logical_or()` und `logical_not()`. 

Reviewer: ```suggestion
erfordern. Stattdessen können die bit-weisen Operatoren `&`, `|`, `~` oder 
die entsprechenden Funktionen `logical_and()`, `logical_or()` und `logical_not()`
verwendet werden. 
```

Eingebaute Python-Funktionen wie `min()` und `max()` werden nicht unterstützt, 

> verwenden Sie stattdessen `fmin()` und `fmax()`.

Reviewer: You use of "Sie"

```suggestion
kann stattdessen `fmin()` und `fmax()` verwendet werden.
```
